bucket/ffmpeg-shared.json

```diff
@@ -10,10 +10,12 @@
             "extract_dir": "ffmpeg-5.0-full_build-shared"
         }
     },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\fonts\")) { New-Item \"$dir\\fonts\" -ItemType Directory | Out-Null }",
```

Comment:
```suggestion
```
*pre_install* is actually not needed, because *Scoop* will treat items in persist as a folder by default, if it does not exist in the archive.